tests/framework/extensions/prime/primechecks.go

```diff
@@ -0,0 +1,83 @@
+package primechecks
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	client "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+	"github.com/rancher/rancher/tests/framework/extensions/rancherversion"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	PodResourceSteveType = "pod"
+	rancherImage         = "rancher"
+)
+
+// CheckUIBrand checks the UI brand of Rancher Prime. If the Rancher instance is not Rancher Prime, the UI brand should be blank.
+func CheckUIBrand(client *rancher.Client, isPrime bool, rancherBrand *client.Setting, brand string) error {
+	if isPrime && brand != rancherBrand.Value {
+		return fmt.Errorf("ERROR: Rancher Prime UI brand %s does not match defined UI brand %s", rancherBrand.Value, brand)
```

Comment:
Nice! This is the last nit, and I'm done. Errors in golang should be in lowercase (go lint style standards), so if you just change it toe "error: ..." it's good to go

Reply:
Oh that's interesting, I did not know that, thanks! Updated it.